CHANGELOG.md

```diff
@@ -62,9 +63,6 @@
 - Editor: add completions from included modules. https://github.com/rescript-lang/rescript/pull/7515
 - Add `-editor-mode` arg to `bsc` for doing special optimizations only relevant to the editor tooling. https://github.com/rescript-lang/rescript/pull/7541
 
-#### :boom: Breaking Change
```

Comment:
@cknitt there is a behaviour change here.
Code compiled with alpha 14 will behave different at runtime after this change